src/App.js

```diff
@@ -1,18 +1,19 @@
-import React, { Component } from 'react';
-import { Provider } from 'react-redux';
-import store from './redux/store';
-import './App.scss';
-import Router from './Router';
+import React, { Component } from "react";
+import { Provider } from "react-redux";
+import store from "./redux/store";
+import "./assets/styles/App.scss";
+// import Router from "./Router";
+import SignupComponent from "./app/Signup/container";
 
 class App extends Component {
   render () {
     return (
       <Provider store={store}>
         <div className="App">
-          <Router />
+          <SignupComponent state={store} />
```

Comment:
instead of commenting out the router you could just wrap the children inside the router
e.g.
```
<provider>
  <router>
     <div className="App">
       <SignupComponent/>
      </div>
  </router>
</provider>
```